flexmeasures/ui/templates/crud/account.html

```diff
@@ -362,6 +401,19 @@ <h3>Assets</h3>
 
     // Collect form data
     const formData = new FormData(event.target);
+
+    // Collect keys of empty fields
+    const emptyFields = [];
+
+    for (var pair of formData.entries()) {
+      if (pair[1] === "") {
+        emptyFields.push(pair[0]);
+      }
+    }
+
+    // Remove the empty fields
```

Comment:
Why should we do that? Good for users to see what they could edit.

Reply:
Oh, this is just a part of the process where I clean up the request body before sending it to the API. For instance, if the user doesn't fill in either the color or logo field, formData will return an empty string. So, I remove those empty fields from the request to ensure a cleaner API request.

Reply:
Actually, I tested this and I wanted to remove the colors and logo again. That is not possible anymore with this code.

Reply:
I went back and tried the reset approach, but it didn't work because formData kept converting null values to empty strings, which the API rejected. To fix this, I manually constructed the payload from formData, giving me full control over the data and its types. Now, I can send a null value to the API, which properly sets the field to null and allows the UI to revert to the default settings.